src/Routes/SeasonsController.ts

```diff
@@ -394,32 +394,37 @@ export class SeasonsController extends BaseController {
          * @swagger
          *
          * definitions:
-         *   TextParagraph:
+         *   TextColumn:
          *     type: object
          *     properties:
          *       content:
          *         type: string
-         *       description:
+         *       language:
          *         type: string
+         *     required: [content, language]
          *
-         *   TextContent:
+         *   TextParagraph:
          *     type: object
          *     properties:
-         *       arabicText:
-         *         schema:
-         *           type: array
-         *           items:
-         *             $ref: '#/definitions/TextParagraph'
-         *       copticText:
+         *       columns:
          *         schema:
          *           type: array
          *           items:
-         *             $ref: '#/definitions/TextParagraph'
-         *       englishText:
+         *             $ref: '#/definitions/TextColumn'

```

Comment:
@behoyh for some reason swagger is not showing this correctly.  I want to say that TextParagraph contains a property which is an array of TextColumn.  eg:

interface TextProperty {
  columns: TextColumn[];
}

How do I correct the syntax?

Reply:
I'm not sure off the top of my head but I can look into it hopefully soon